### PR TITLE
update border color

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes [107](https://github.com/brightlayer-ui/angular-themes/issues/107) .

#### Changes proposed in this Pull Request:
- fix mat expansion header border 

#### Screenshots:
image

Showcase demo:
Screenshot 2022-04-07 at 12 36 51 PM

#### To Test:
Angular-design-pattern
- /lists/status-list
- /lists/inline-local-actions

Angular showcase demo:
/templates/dashboard

### CODE REVIEW

**Comment by ektasawant** [comment]
https://github.com/brightlayer-ui/angular-showcase-demo/blob/dev/src/app/pages/templates/dashboard/dashboard.component.html#L169 need to remove this line to fix double border (this issue is only in showcase demo).

File: _blueTheme.scss [hunk 1]
```diff
@@ -514,4 +514,13 @@
     th.mat-header-cell {
         color: map-get($foreground, text);
     }
+
+    /* Mat Expansion Panel Header */
+    .mat-expansion-panel {
+        .mat-expansion-panel-header {
+            &.mat-expanded {
+                border-bottom: 1px solid rgba(map-get(blui.$blui-black, 500), 0.12);
```
Comment (emclaug2) [inline _blueTheme.scss:522]: `1px solid rgba(map-get($foreground.divider));`

File: _darkTheme.scss [hunk 2]
```diff
@@ -648,4 +648,13 @@
     th.mat-header-cell {
         color: map-get($foreground, text);
     }
+
+    /* Mat Expansion Panel Header */
+    .mat-expansion-panel {
+        .mat-expansion-panel-header {
+            &.mat-expanded {
```
Comment (emclaug2) [inline _darkTheme.scss:655]: `border-bottom: 1px solid map-get($foreground, divider);`

**Comment by emclaug2** [comment]
The changes work, but since we've already defined this color in both palettes, we can use the palette here instead.  Also can you update the changelog?

**Review by emclaug2 — APPROVED** [review]

**Pull request merged**